src/deepsparse/engine.py

```diff
@@ -55,8 +61,10 @@ def _model_to_path(model: Union[str, Model, File]) -> str:
     if not model:
         raise ValueError("model must be a path, sparsezoo.Model, or sparsezoo.File")
 
-    if isinstance(model, str):
-        pass
+    if isinstance(model, str) and model.startswith("zoo:"):
+        if sparsezoo_import_error is not None:
+            raise sparsezoo_import_error
+        model = Zoo.load_model_from_stub(model).onnx_file.downloaded_path()
```

Comment:
NIT: might make sense to have this overwrite the model to the Model instance returned just from load_model_from_stub and then add a new line and start the if statement again. That way we keep from duplicating the onnx_file.downloaded_path() portion in the event we change the zoo model flow and resolving a runnable model in the future